config/locales/en.yml

```diff
@@ -307,7 +307,7 @@ en:
     attachments: "Attachments"
     author: "Author"
     base: "General Error:"
-    blocks_ids: "Blocks"
+    blocks_ids: "IDs of blocked work packages"
```

Comment:
Just for reference: `openproject-backlogs` defines a localization string called `label_blocks_ids` which translates to
> Blocks (IDs)

You might want to harmonize that in one direction or another.